docker-compose.yml

```diff
@@ -1,11 +1,17 @@
 version: "3"
 
 services:
+  # replace with normal redis?
   redis:
     image: docker.io/redislabs/redisearch:2.4.0
     ports:
       - "6379:6379"
 
+  meilisearch:
```

Comment:
We need to make sure, that this service is not exposed to the web in the staging/production deployments